presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/PlanBuilder.java

```diff
@@ -468,9 +470,22 @@ public JoinNode join(
         return new JoinNode(idAllocator.getNextId(), type, left, right, criteria, outputSymbols, filter, leftHashSymbol, rightHashSymbol, Optional.empty());
     }
 
-    public UnionNode union(List<? extends PlanNode> sources, ListMultimap<Symbol, Symbol> outputsToInputs, List<Symbol> outputs)
+    public UnionNode union(List<PlanNode> sources, ListMultimap<Symbol, Symbol> outputsToInputs)
```

Comment:
please put `sources` a the end. You might also consider using varargs instead of list for `sources`